builtin/providers/aws/resource_aws_route53_record.go

```diff
@@ -380,6 +380,14 @@ func resourceAwsRoute53RecordBuildSet(d *schema.ResourceData, zoneName string) (
 			HostedZoneID:         aws.String(alias["zone_id"].(string)),
 		}
 		log.Printf("[DEBUG] Creating alias: %#v", alias)
+	} else {
+		if _, ok := d.GetOk("ttl"); !ok {
+			return nil, fmt.Errorf(`provider.aws: aws_route53_record: %s: "ttl": required field is not set`, d.Get("name").(string))
```

Comment:
> Sorry for being picky here (I know it suppose to be just a temp fix :smiley: ), but:
nah, It's alright :dancer:  

> Any specific reason why do use back-ticks instead of double quotes for escaping the string here?

Because the name of the attribute is already using double quotes. I'm open to suggestions, though.

> I think provider.aws is now a bit superfluous since the resource is called aws_route53_record (= it's IMO obvious that it's coming from AWS provider).

I was only following the convention currently in used by Terraform's error messages for missing required attributes. Do you want me to remove it?